### DISCUSSION TITLE
How to upgrade 2.12 to 2.13?

### DISCUSSION THREAD

**Discussion by softshipper** [quarkusio/quarkus#28341] [Q&A] [answered]
Hi all,
At the moment I am using Quarkus 2.12 and I would like to upgrade to version 2.13. 
The question is, how to do the upgrade.
I am using the build tool Gradle.

Best regards,
Anu

**Comment by maxandersen** [comment, marked as answer]
Change to the version 2.13 of the quarkus platform in build.gradle / gradle.properties.

**Reply by softshipper** [reply]
Thx a lot.
